design_linked_list.py

```diff
@@ -0,0 +1,73 @@
+class Node:
+    def __init__(self, val = 0, next = None):
+        self.val = val
+        self.next = next
+        
+class MyLinkedList:
+
+    def __init__(self):
+        self.head = Node()
+        self.length = 0
+
+    def get(self, index: int) -> int:
+        if index >= self.length:
+            return -1
+        
```

Comment:
A  variable name cur might be a little confusing, even tho its used by many programmers. 
May be if you could change it to something else. `cur = self.head.next`